assets/src/application/ui/layout/espressoDropdownMenu/style.scss

```
@@ -20,21 +20,31 @@
 }
 
 .components-dropdown-menu__menu {
+	padding: var(--ee-padding-smaller) var(--ee-padding-smaller) var(--ee-padding-small);
 	width: 100%;
-	padding: 7px 0;
-	font-size: 13px;
-	line-height: 1.4;
 
 	.components-dropdown-menu__menu-item,
 	.components-menu-item {
-		background-color: transparent !important;
 		border: none !important;
 		box-shadow: unset !important;
 		cursor: pointer;
-		margin: 0 !important; // important was needed because it is conflicting with EspressoButton styles
+		font-size: var(--ee-font-size-small);
+		font-weight: inherit;
+		margin: 0 0 5px !important; // important was needed because it is conflicting with EspressoButton styles
 		outline: none;
-		padding: 6px;
+		padding: 0 !important;
 		width: 100% !important; // important was needed because it is conflicting with EspressoButton styles
+
+		&:focus:not(:disabled):not([aria-disabled='true']):not(.is-secondary) {
+			color: #191e23;
+			background-color: var(--ee-color-grey-14);
```

```suggestion
			color: var(--ee-text-on-grey-14);
			background-color: var(--ee-color-grey-14);
```

You can get a [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) compliant text color for any background color var, simply by replacing the word `color` with `text-on`

examples: 

background color | text color
--- | ---
`--ee-` `color` `-grey-14` | `--ee-` `text-on` `-grey-14`
`--ee-` `color` `-primary` | `--ee-` `text-on` `-primary`
`--ee-` `color` `-purple` | `--ee-` `text-on` `-purple`
`--ee-` `color` `-yellow-super-low-contrast` | `--ee-` `text-on` `-yellow-super-low-contrast`

---

very handy and smart